src/main/java/com/ea/restaurant/util/GrpcUtil.java

```diff
@@ -0,0 +1,14 @@
+package com.ea.restaurant.util;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+
+public class GrpcUtil {
+
+  public static String[] getCredentialsFromBasicAuthToken(String authorizationValue) {
+    var base64Credentials = authorizationValue.substring("basic".length()).trim();
```

Comment:
Use latest java features:
```
Optional..of(authorization).map(...).map(...).orElseThrow(...)
```
Use `Stream.of()` in order to convert arrays to list.

Reply:
Done.